### PR TITLE
added rebalancing to getAdddresses method

### PR DESCRIPTION
**Pull request by simusud**
removed setMaxSellAmount action from _action dict and added rebalancing address in getAddresses methods in data_objects.py
added rebalancing interface in interfaces.py
added units tests for changes

### CODE REVIEW

File: core_contracts/governance/data_objects.py [hunk 1]
```diff
@@ -28,7 +28,7 @@ class VoteActions(object):
     def __init__(self, db: IconScoreDatabase, gov: IconScoreBase) -> None:
         self._db = db
         self._gov = gov
-        self._actions = {
+        self._actions = [{
```
Comment (dbrehmer) [inline core_contracts/governance/data_objects.py]: This looks like a mistake. self._actions is now a one-element list.

In issue #407 we need to change the actions in defineVote to a list.

**Review by dbrehmer — CHANGES_REQUESTED** [review]
I'm not sure if this was also intended to address issue #407. If so, it needs a change for that. Otherwise there was a typo.

File: core_contracts/governance/data_objects.py (hunk 1 above)
Comment (dbrehmer) [reply core_contracts/governance/data_objects.py]: Ah, I see it is addressed in the other PR, so here the list is just a typo.

Comment (simusud) [reply core_contracts/governance/data_objects.py]: Oh.. yes it must be a typo. I will fix that .

**Review by dbrehmer — APPROVED** [review]
Looks good.

**Pull request merged**